packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.8.6
```

Comment:
Please revert the changes to `image_picker`; there's no need to update the app-facing dependencies for a bugfix in an implementation package.